### ISSUE TITLE
Dynamic description & keywords

### ISSUE THREAD

**Issue by wewerica** [lionix-team/seo-manager#3]
Hello,
is there any possibility to make dynamic description and keywords for blog posts for example?

**Comment by karakhanyans** [comment]
Yes, of course, this package is for that kind of stuff, for the dynamic meta data.
Can I help you to do that?

**Comment by wewerica** [comment]
Great :)
I found how to set dynamic title but don't know how to set dynamic description & keywords :(
I have this route for pages and I would like to set different description and keywords for each page.

![image](https://user-images.githubusercontent.com/1906537/53425218-e93e1f00-39e4-11e9-865d-8fa13905aae8.png)

I was able to set OG meta tags but not description and keywords meta tags.

Thanks

**Comment by karakhanyans** [comment]
@Wewerica you can have different keyword and description for each route, but at this moment you can't set it like you set dynamic title, it can be different for every page/route but not for every post. 
Instead of that now you can use OG meta. I will add this functionality, you are right, it's useful. 
Thank you very much @Wewerica. I will push that functionality asap.

**Comment by wewerica** [comment]
@karakhanyans Great. Thanks
It would be great if you could add other meta tags as well, it would be usefull.
I know I’m asking too much 🙁

**Comment by karakhanyans** [comment]
@Wewerica you are not asking too much, I want this package to be useful and want to add a lot of functionalities to it. Thanks for your suggestion here.

**Comment by wewerica** [comment]
@karakhanyans 
hi
can you push functionality for me to test it? :D

thanks
